src/EditorFeatures/Core/Options/LegacyGlobalOptionsWorkspaceService.cs

```diff
@@ -23,22 +23,22 @@ internal sealed class LegacyGlobalOptionsWorkspaceService : ILegacyGlobalOptions
         private readonly CodeActionOptionsStorage.Provider _provider;
 
         private static readonly Option2<bool> s_generateOverridesOption = new(
-            "GenerateOverridesOptions_SelectAll", defaultValue: true);
+            "dotnet_select_all_when_generate_overrides", defaultValue: true);
 
         private static readonly PerLanguageOption2<bool> s_generateOperators = new(
-            "GenerateEqualsAndGetHashCodeFromMembersOptions_GenerateOperators",
+            "dotnet_generate_equals_and_get_hash_code_from_members_when_generate_operators",
             defaultValue: false);
 
         private static readonly PerLanguageOption2<bool> s_implementIEquatable = new(
-            "GenerateEqualsAndGetHashCodeFromMembersOptions_ImplementIEquatable",
+            "dotnet_generate_equals_and_get_hash_code_from_members_when_implement_iequatable",
             defaultValue: false);
 
         private static readonly PerLanguageOption2<bool> s_addNullChecks = new(
-            "GenerateConstructorFromMembersOptions_AddNullChecks",
+            "dotnet_add_null_checks_when_generate_constructor_from_members",
             defaultValue: false);
 
         internal static readonly PerLanguageOption2<bool> AddNullChecksToConstructorsGeneratedFromMembers = new(
-            "GenerateConstructorFromMembersOptions_AddNullChecks",
+            "dotnet_add_null_checks_when_generate_constructor_from_members",
```

Comment:
>dotnet_add_null_checks_when_generate_constructor_from_members [](http://example.com/codeflow?start=13&length=61)

Is this a duplicate option definition? What's the difference between this one and the previous one?

Reply:
I also feel puzzled when I rename this option.
It's only used in a unit test : )

Reply:
This one is `internal` and the previous one is `private`.
I can try to remove this, and see we will break any IVT friends.

Reply:
I will leave this comment open